index.html

```diff
@@ -38,6 +38,8 @@
 						<div class="tabs-link" data-tab="2">Tab 2</div>
 						<div class="tabs-link" data-tab="3">Tab 3</div>
 						<div class="tabs-link" data-tab="4">Tab 4</div>
+						<div class="tabs-link" data-tab="3">Tab 5</div>
+						<div class="tabs-link" data-tab="4">Tab 6</div>
```

Comment:
The `data-tab` attribute values for Tab 5 and Tab 6 are not set to their respective tab numbers (set to `3` and `4` respectively - same as their counterparts above) - these tabs will work correctly once you edit the `data-tabs` values accordingly.